
From: Christian Borntrager <christian@borntraeger.net>

Two scsi drivers do not compile on systems without ISA/PCI.  Therefore
allyesconfig breaks on hardware like s390.  

Signed-off-by: Christian Bornträger <linux-kernel@borntraeger.net>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/scsi/Kconfig |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/scsi/Kconfig~add-bus-dependencies-to-two-scsi-drivers drivers/scsi/Kconfig
--- 25/drivers/scsi/Kconfig~add-bus-dependencies-to-two-scsi-drivers	2004-08-04 23:28:16.725629576 -0700
+++ 25-akpm/drivers/scsi/Kconfig	2004-08-04 23:28:16.741627144 -0700
@@ -329,7 +329,7 @@ source "drivers/scsi/aic7xxx/Kconfig.aic
 
 config SCSI_AIC7XXX_OLD
 	tristate "Adaptec AIC7xxx support (old driver)"
-	depends on SCSI
+	depends on (ISA || EISA || PCI ) && SCSI
 	help
 	  WARNING This driver is an older aic7xxx driver and is no longer
 	  under active development.  Adaptec, Inc. is writing a new driver to
@@ -604,7 +604,7 @@ config SCSI_EATA_MAX_TAGS
 
 config SCSI_EATA_PIO
 	tristate "EATA-PIO (old DPT PM2001, PM2012A) support"
-	depends on SCSI
+	depends on (ISA || EISA || PCI) && SCSI
 	---help---
 	  This driver supports all EATA-PIO protocol compliant SCSI Host
 	  Adapters like the DPT PM2001 and the PM2012A.  EATA-DMA compliant
_
